api/src/main/java/org/openmrs/serialization/SimpleXStreamSerializer.java

```diff
@@ -40,39 +47,91 @@
  * </code>
  *
  */
+@Component("simpleXStreamSerializer")
 public class SimpleXStreamSerializer implements OpenmrsSerializer {
+
+	protected static final Logger log = LoggerFactory.getLogger(SimpleXStreamSerializer.class);
 	
 	// cached xstream object
-	public XStream xstream = null;
+	private volatile XStream xstream;
```

Comment:
Can we rename this to just clearly distinguish between this private member variable and the xstream argument passed into the static methods below?

Reply:
Renamed arguments in methods instead.